vrs/vrs_annotation_batch.py

```diff
@@ -349,19 +367,19 @@ def main(args):
         "--disk-size",
         help="Amount of disk (GB) to allocate to each Hail Batch job.",
         type=float,
-        default=6 
+        default=None,
```

Comment:
couldn't comment on exact line, but help text for "version" arg should end with a period, and for "working_bucket" arg "Name of GCP Bucket to mount for access." should change to somethings like "Name of GCP Bucket to which intermediate files and final test results should be output."

Reply:
Ends in a period now and working_bucket has been revised. Going ahead and resolving